services/QuillLMS/engines/evidence/lib/evidence/evidence/check.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Evidence
+  module Check
+    ALL_CHECKS = [
+      Prefilter,
+      RegexSentence,
+      Opinion,
+      Plagiarism,
+      AutoML,
+      RegexPostTopic,
+      Grammar,
+      Spelling,
+      RegexTypo
+    ]
+
+    FALLBACK_RESPONSE = {
+      feedback: "Thank you for your response.",
+      feedback_type: "autoML",
+      optimal: true,
+    }
+
+    def self.feedback(entry, prompt, previous_feedback)
```

Comment:
Pure style question, but does this make more sense as `get_feedback` or `fetch_feedback`?  If a method might create significant work/delay in a workstream, I like to give it a name that implies that it will do some work.  Not critical, just a preference.

Reply:
I was debating using `get_feedback` here, and given you would find that clearer, I've changed the name to that.